charts/rancher-wins-upgrader/values.yaml

```diff
@@ -26,18 +23,18 @@ winsConfigs:
     proxy: rancher_wins_proxy
     whiteList:
       processPaths:
-        - {{ .Values.prefixPath }}etc\rancher\wins\wins-upgrade.exe
-        - {{ .Values.prefixPath }}etc\windows-exporter\windows-exporter.exe
-        - {{ .Values.prefixPath }}etc\wmi-exporter\wmi-exporter.exe
-        - {{ .Values.prefixPath }}etc\kubernetes\bin\kube-proxy.exe
-        - {{ .Values.prefixPath }}etc\kubernetes\bin\kubelet.exe
-        - {{ .Values.prefixPath }}etc\nginx\nginx.exe
-        - {{ .Values.prefixPath }}opt\bin\flanneld.exe
+        - {{ .Values.global.cattle.rkeWindowsPrefixPath }}etc/rancher/wins/wins-upgrade.exe
+        - {{ .Values.global.cattle.rkeWindowsPrefixPath }}etc/windows-exporter/windows-exporter.exe
+        - {{ .Values.global.cattle.rkeWindowsPrefixPath }}etc/wmi-exporter/wmi-exporter.exe
+        - {{ .Values.global.cattle.rkeWindowsPrefixPath }}etc/kubernetes/bin/kube-proxy.exe
+        - {{ .Values.global.cattle.rkeWindowsPrefixPath }}etc/kubernetes/bin/kubelet.exe
+        - {{ .Values.global.cattle.rkeWindowsPrefixPath }}etc/nginx/nginx.exe
+        - {{ .Values.global.cattle.rkeWindowsPrefixPath }}opt/bin/flanneld.exe
```

Comment:
Doesn't flipping the slashes cause an issue with the wins config? this would be exactly what is deposited in the wins config, so I had initially designed this to match what we put in our bootstrap script in https://github.com/rancher/rancher/blob/29e8af8818e56dc35e1bac239685803f62baa4ba/package/windows/bootstrap.ps1#L280-L293.

If we want to do forward slashes, we may need to modify the tpl call here to do ` {{ tpl .config $ | replace "/" "\\" indent 4 }}`:

https://github.com/rancher/wins/blob/72fac8616d77b23e7ce9c325622718e6a9857679/charts/rancher-wins-upgrader/templates/configmap.yaml#L10